### PR TITLE
[state-sync] cleaned up state-sync structured logging

### PR DESCRIPTION
**Pull request by gregnazario**
Replaced most debug logging with display logging accordingly, and
attempted to reduce the likelihood of spewing out large amounts of data
into structured logging.

It's really up to those who are familiar with it to tell me whether the LedgerInfo, which I've mostly stopped logging because of its extremely large overhead is required to be logged and whether these new amounts are the right level.

### CODE REVIEW

**Comment by zekun000** [comment]
I think log the Display format is good enough, no one will look at the signatures in the debug format.

**Comment by gregnazario** [comment]
> I think log the Display format is good enough, no one will look at the signatures in the debug format.

Here's the problem.  The structured log converter takes every field that's in the `debug!("{},{}",a,b)` and runs the debug printer on it.  Which is why in consensus it was spewing large amounts of data in structured logging, even though it had a display handler.

tl;dr We have a display handler, if the debug doesn't provide us more useful info just override it with the display.
